### PR TITLE
Cleanup linter errors

### PR DESCRIPTION
**Pull request by johansglock**
Please pay careful attention to extra errors returned that weren't returned previously, that's the only thing I could see having impact here.

### CODE REVIEW

**Comment by johansglock** [comment]
@ddn0 @m-i-sadowski @chr1sgrant 
I've done a major cleanup to have all logic linter checks pass, a few places I didn't deem worth fixing which are either covered by '// nolint' or excluded from the gometalinter command.

This enables a first pass for consistent codebase. Since this rewrites a huge amount of small logic cases, the diff requires really careful checking.

File: microArch/scheduler/liquidhandling/setupagent.go [hunk 1]
```diff
@@ -236,10 +242,13 @@ func BasicSetupAgent(ctx context.Context, request *LHRequest, params *liquidhand
...
 
-			err = params.AddTipWaste(waste)
+			if err != nil {
+				return nil, wtype.LHError(wtype.LH_ERR_OTHER, fmt.Sprintf("error for liquid handler of model %s: %s", params.Model, err))
+			}
 
+			err = params.AddTipWaste(waste)
 			if err != nil {
-				return nil, wtype.LHError(wtype.LH_ERR_OTHER, fmt.Sprintf("Error for liquid handler of model %s: %s", params.Model, err))
+				return nil, wtype.LHError(wtype.LH_ERR_OTHER, fmt.Sprintf("error adding tip waste: %s", err))
```
Comment (m-i-sadowski) [inline microArch/scheduler/liquidhandling/setupagent.go]: Can we reinstate the model as well here?

File: antha/AnthaStandardLibrary/Packages/Optimization/main/optimize2.go [hunk 2]
```diff
@@ -1,125 +1,5 @@
 package main
 
-import (
```
Comment (m-i-sadowski) [inline antha/AnthaStandardLibrary/Packages/Optimization/main/optimize2.go]: I'm concerned we're losing a lot of potentially useful code here and below. Some of it is WIP stuff but it'd be preferable to keep it in, perhaps with a trivial main function?

Comment (johansglock) [reply antha/AnthaStandardLibrary/Packages/Optimization/main/optimize2.go]: Added it back as a comment

**Comment by haydnKing** [comment]
@johansglock @m-i-sadowski @ddn0 @chr1sgrant There's still two bundles which will need fixing for antha-tester to work in repos, but how do we feel about merging these in?

**Pull request merged**